test/test_operators.py

```diff
@@ -284,6 +284,24 @@ def linear_no_bias(input, weight):
         ref_out_no_bias, tvm_out_no_bias = self.runBoth(linear_no_bias, input, weight)
         assert torch.allclose(ref_out_no_bias, tvm_out_no_bias, rtol=0.01, atol=0.01)
 
+    @TVMTest.given(
+        shape=TVMTest.rand_shape(rank=2, min_dim=4),
+    )
+    def test_concat(self, shape):
+        input1 = torch.rand(shape)
+        input2 = torch.rand(shape)
+        # import pdb
+        # pdb.set_trace()
+        def concat(x1, x2):
+            return torch.cat((x1, x2), 0)
+
+        import torch_tvm
+        torch_tvm.enable()
+        # test concat
+        scripted_concat = torch.jit.script(concat)
+        concat_graph = scripted_concat.graph_for(input1, input2)
+        FileCheck().check("prim::FusedConcat").check_not("prim::ListConstruct").check_not("aten::cat").run(str(concat_graph))
```

Comment:
This seems to be the test not for concat but whether ListConstruct and cat are fuse. We should name it such.